mongo/assets/configuration/spec.yaml

```diff
@@ -158,16 +158,18 @@ files:
             - name: database
               type: string
             - name: query
-              type: string
-            - name: fields
               type: object
```

Comment:
I'm not sure if query should be of type oneOf(object, str) but seems we don't support oneOf